view/src/extras/dealStepper.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Stepper from "@material-ui/core/Stepper";
+import Step from "@material-ui/core/Step";
+import StepLabel from "@material-ui/core/StepLabel";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import Surplus from "../components/surplus";
+import FoodBanks from "../components/foodbanks";
+import DealForm from "../extras/dealForm";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+  },
+  button: {
+    marginRight: theme.spacing(1),
+  },
+  instructions: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+/** Defines the steps in the Stepper (each is a form) */
+function getSteps() {
+  return [
+    "Select a Surplus Object",
+    "Match with a Food Bank",
+    "Fill out Deal Logistics",
+  ];
+}
+
+export default function HorizontalLinearStepper(props) {
+  const classes = useStyles();
+  // Stepper states
+  const [activeStep, setActiveStep] = React.useState(0);
+
+  const steps = getSteps();
+
+  /** Gets the content to display at each step */
+  const getStepContent = (step) => {
+    switch (step) {
+      case 0:
+        return <Surplus inSurplus={true} alert={props.alert} />;
+      case 1:
+        return <FoodBanks inSurplus={true} alert={props.alert} />;
+      case 2:
+        return <DealForm alert={props.alert} />;
+      default:
+        return "Unknown step";
+    }
+  };
+
+  /** Sets the last step (so formatting can change on last step) */
+  const isLastStep = (step) => {
+    return step === 2;
+  };
+
+  /** Moves to the next step (page) in the Stepper */
+  const handleNext = () => {
+    // Confirm that selected (needs to be connected to child)
+    if (false) {
```

Comment:
um.

this'll never run.

Reply:
It is a placeholder so that I remember to connect it to the child (it is a check to ensure that the user selects a surplus object before moving to the next page of the stepper)

Reply:
I'll cut it an leave a comment

Reply:
Got it. Yeah, this would be better as a comment.